shema.sql

```diff
@@ -20,3 +20,18 @@ CREATE TABLE LABEL (
   book_id INT,
   FOREIGN KEY (book_id) REFERENCES item(id) ON DELETE CASCADE
 );
+
+CREATE TABLE GAME (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  multiplayer VARCHAR(50) NOT NULL,
+  last_player_at VARCHAR(50) NOT NULL,
+  FOREIGN KEY (id) REFERENCES item(id) ON DELETE CASCADE
```

Comment:
- [ ] According to our UML diagram, I think it would be `Date ` instead of Varchar.
- [ ] And you can add  `item_id `I NT,
  FOREIGN KEY `(item_id)` REFERENCES item(id) ON DELETE CASCADE
```suggestion
   last_player_at Date NOT NULL,
   item_id `INT,
  FOREIGN KEY `(item_id)` REFERENCES item(id) ON DELETE CASCADE
```